docs/package.json

```diff
@@ -21,7 +21,6 @@
     "@nuxtjs/seo": "^2.0.0-rc.10",
     "@vueuse/core": "^10.10.1",
     "@vueuse/nuxt": "^10.10.1",
-    "nuxt": "npm:nuxt-nightly@latest",
```

Comment:
### Note

corrected what I was talking this.

https://github.com/nuxt/scripts/pull/93#pullrequestreview-2115363678